bundles/http/Citrix/Citrix-Virtual-Apps-and-Desktops-service/7687284a-09a1-4494-b33c-f6bfa0932656.js

```diff
@@ -0,0 +1,671 @@
+                                'session_app_in_use',
+                                sessions_applications_in_use,
+                            )
+                        }
+                    }
+                    params.dataStore.save('session', sessions)
+                } else {
+                    console.log(
+                        'WARNING: Endpoint Sessions - Response is empty or in wrong format.',
+                    )
+                }
+            } catch (error) {
+                console.error(error)
+            }
+        } while (continuationToken)
+    }
+
+    console.log('Sync of Sessions finished.')
+}
+
+async function getSiteIds({client, integrationParameters}) {
+    console.log('Started getting Site Ids.')
+
+    let response = await client.fetch('/me', {
+        headers: {
+            'Content-Type': 'application/json',
+            'Citrix-CustomerId': integrationParameters.customerId,
+        },
+    })
+
+    if (!response.ok) {
+        throw new Error(`Getting Site IDs failed(${response.status}: ${response.statusText})`)
+    }
+
+    response = await response.json()
+
+    const siteIds = []
+
+    try {
+        response.Customers.forEach(customer =>
+            customer.Sites.forEach(site => siteIds.push(site.Id)),
+        )
+    } catch (err) {
+        console.error(err)
+    }
+
+    console.log('Finished getting Site IDs')
+
+    return siteIds
+}
+
+async function request(
+    siteId,
+    {client, integrationParameters},
+    entity,
+    continuationToken,
+    fields,
+    body,
+) {
+    let path = `${siteId}/${entity}/$search?fields=${fields}&limit=${limit}`
+
+    if (continuationToken) {
+        path = `${path}&continuationToken=${continuationToken}`
+    }
+    const response = await client.fetch(path, {
+        method: 'POST',
+        body: body,
+        headers: {
+            'Content-Type': 'application/json',
+            'Citrix-CustomerId': integrationParameters.customerId,
+        },
+    })
+
+    if (!response.ok) {
+        throw new Error(
+            `Endpoint ${entity} - Request failed(${response.status}: ${response.statusText})`,
+        )
+    }
+    console.log(entity + ' - response received, status: ' + response.status)
+
+    return response.json()
+}
+
+function flatJson(json, oName) {
+    if (json[oName] != undefined) {
+        Object.entries(json[oName]).map(([key, value]) => (json[oName + '_' + key] = value))
+        delete json[oName]
+    }
+    return json
+}
+
+integration.define({
+    synchronizations: [
+        {
+            name: 'sync',
+            fullSyncFunction: fullSync,
+            incrementalSyncFunction: incrementalSync,
+        },
+    ],
+    model: {
+        tables: [
+            {
+                name: 'machine',
+                columns: [
+                    {
+                        name: 'AgentVersion',
+                        type: 'STRING',
+                        length: 255,
```

Comment:
Is it enough?
I suppose it is meant as a browser agent. It sometimes could be very long, but not sure if it is important if it will be shorter :)

Reply:
It looks usually just like this `1808.2.0.19074` - just some version of the Agent. 🤷‍♂️